src/components/common/status-chip/StatusChip.tsx

```diff
@@ -17,9 +17,14 @@ const StatusColourMap = new Map([
 ]);
 
 type Props = {
+  className?: string;
```

Comment:
I've been seeing more of this question mark notation around. Does it mean if className is provided it'll be a string?

Reply:
this means it's an optional prop! its value will default to its value in `defaultProps` defined below